templates/Containerisation/helm/helm-deploy.yaml

```diff
@@ -0,0 +1,46 @@
+parameters:
+  - name: helmVersion
+    type: string
+    default: 'latest'
+  - name: azureSubscription
+    type: string
+    default: ""
+  - name: kubernetesCluster
+    type: string
+    default: "ss-dev-01"
+  - name: acrName
+    type: string
+    default: "sdshmctspublic"
+  - name: namespace
+    type: string
+    default: "vh"
+  - name: chartName
+    type: string
+  - name: chartPath
+    type: string
+  - name: imageTag
+    type: string
+    default: "$(Build.BuildNumber)"
+
+steps:
+  - bash: |
+        ls -ltr ${{ parameters.chartPath }}
+        echo "general filecheck"
+        ls -ltr . 
+
+  - task: HelmDeploy@0
+    displayName: Helm upgrade
+    inputs:
+      connectionType: 'Azure Resource Manager'
+      azureSubscriptionEndpoint: '${{ parameters.azureSubscription }}'
+      azureResourceGroup: ${{ parameters.kubernetesCluster }}-rg 
+      kubernetesCluster: ${{ parameters.kubernetesCluster }}-aks
+      command: upgrade
+      chartType: filepath
+      chartPath: ${{ parameters.chartPath }}/${{ parameters.chartName }}-${{ parameters.imageTag }}.tgz
```

Comment:
Would you want to leave it to the user to either pass a tgz file or just a path to the Charts.yaml?
This would assume they have packaged the release.

Reply:
I think it has to be a tgz file? See this run error https://dev.azure.com/hmcts/Video%20Hearings/_build/results?buildId=212474&view=logs&j=cb886c1e-b3ce-5825-a7e1-7f1f61b50c52&t=90490d3e-cdf4-5c04-068c-c304bf6d6aa0&l=12

Reply:
That will be as you passed 
`/home/vsts/work/1/s/vh-bookings-api/charts/vh-bookings-api/Chart.yaml`
instead of
`/home/vsts/work/1/s/vh-bookings-api/charts/vh-bookings-api`

![image](https://user-images.githubusercontent.com/86669094/154683050-79785472-49d4-47ca-a25e-6bd32de1e72c.png)